app/assets/stylesheets/layouts/dark.scss

```diff
@@ -87,58 +189,58 @@ $ODD_COLOR: #474747;
 /* TODO: hover, click */
 
 /* Darken input boxes */
-$INPUT_BG: #4D4D4D; /* for generic input boxes all over the place */
-$INPUT_BORDER: #666666;
-$INPUT_PLACEHOLDER: #888888;
-$MCE_PANEL_BG: #404040; /* MCE header & footer background */
-$SELECT2_HIGHLIGHT: #616161;
-$SELECT2_SELECTED: #454545;
-$SELECT2_SEARCH: $SELECT2_HIGHLIGHT;
-$FONT_COLOR: #BBB; /* TODO: standardize font color */
+$font_color_input: #BBB; /* TODO: standardize font color */
```

Comment:
todo?

Reply:
what was the todo for, that was already there

Reply:
oh, I didn't notice that. um. I don't recall. you can leave it. xD